misc/testerina/modules/report-tools/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@ballerina/testerina-report",
-    "version": "2.0.0-Preview1-M3",
+    "version": "2.0.0-SNAPSHOT",
```

Comment:
Do we need this change?

Reply:
Looks like a merge issue, ill remove and go though the changed file list again